apps/engine/src/executeMainThread.ts

```diff
@@ -109,7 +119,10 @@ export const executeMainThread = async () => {
 		return;
 	}
 
-	const argv = await Promise.resolve(argvObject.argv);
+	const argv = {
+		...(await Promise.resolve(argvObject.argv)),
+		'arg:input': userInput,
+	};
```

Comment:
Can you explain in the PR summary why user input is required in this PR. Maybe add a video or images of how this will be useful.